File: FindMyBuddiesProjectServer/FindMyBuddies.Api/Models/UserModel.cs
using System;
using System.Runtime.Serialization;

namespace FindMyBuddies.Api.Models
{
    [DataContract]
    public class UserModel
    {
        [DataMember(Name = "username")]
        public string Username { get; set; }

        [DataMember(Name = "nickname")]
        public string Nickname { get; set; }

        [DataMember(Name = "authCode")]
        public string AuthCode { get; set; }
    }
}

File: FindMyBuddiesProject/FindMyBuddies/src/main/java/com/gercho/findmybuddies/models/UserModel.java
package com.gercho.findmybuddies.models;

/**
 * Created by Gercho on 11/10/13.
 */
public class UserModel {

    private String username;
    private String authCode;
    private String nickname;
    private String sessionKey;

    public UserModel(String username, String authCode, String nickname) {
        this.username = username;
        this.authCode = authCode;
        this.nickname = nickname;
    }

    public UserModel(String username, String authCode) {
        this.username = username;
        this.authCode = authCode;
    }

    public String getUsername() {
        return this.username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getAuthCode() {
        return this.authCode;
    }

    public void setAuthCode(String authCode) {
        this.authCode = authCode;
    }

    public String getNickname() {
        return this.nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

    public String getSessionKey() {
        return sessionKey;
    }

    public void setSessionKey(String sessionKey) {
        this.sessionKey = sessionKey;
    }
}
